service-api/app/src/App/src/Service/Lpa/AddOlderLpa.php

```diff
@@ -34,14 +37,18 @@ public function __construct(
         LpaAlreadyAdded $lpaAlreadyAdded,
         OlderLpaService $olderLpaService,
         ValidateOlderLpaRequirements $validateOlderLpaRequirements,
-        LoggerInterface $logger
+        LoggerInterface $logger,
+        FeatureEnabled $featureEnabled,
+        RestrictSendingLpaForCleansing $restrictSendingLpaForCleansing
```

Comment:
```suggestion
        RestrictSendingLpaForCleansing $restrictSendingLpaForCleansing,
        LoggerInterface $logger,
        FeatureEnabled $featureEnabled
```

Seems like a silly thing but I've been trying to keep 'utility' classes at the end of the constructor and 'doing things' classes at the start.